.agent/src/jinx/state.py

```
@@ -107,120 +95,75 @@ class StateBlock(BaseModel):
-            Dict[str, Any]: The configuration dict, or an empty dict on missing or invalid file.
-        """
+        """Loads and parses the master JINX.yaml configuration state."""
         jinx_path = _resolve_jinx_path()
         if not jinx_path.exists():
-            logger.debug("JINX state file not found at %s. Returning empty state.", jinx_path)
             return {}
         try:
             with open(jinx_path, "r", encoding="utf-8") as f:
                 data = yaml.safe_load(f)
                 return data if isinstance(data, dict) else {}
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)